content/php/concepts/string-functions/terms/ltrim/ltrim.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'ltrim()'
+Description: 'Trims whitespace or specific characters from the left side of a given string.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `ltrim()` function accepts a string as its parameter, removes whitespace from the left side of that string, and then returns it.
+
+Optionally, `ltrim()` can accept a second parameter containing specific characters to remove from the left side of a target string.
+
+## Syntax
+
+```pseudo
+ltrim($string, $characters);
+```
+
+The `$string` parameter gives the function a target string to remove whitespace or specific characters from.
+
+The `$characters` parameter is an optional string that specifies specific characters the function should remove. `ltrim()` will start at the left side of the string and remove any characters that match the ones provided in the `$characters` argument until it finds a character that does not match.
+
+## Codebyte Example
+
+The following example demonstrates the `ltrim()` function with both one and two arguments.
+
+```codebyte/php
```

Comment:
In order for this to work on an HTML page, the `<?php ... ?>` block needs to be wrapped in a `<pre>` element in order to preserve whitespace, otherwise the HTML rendering will remove the whitespace making the output something like this: 
```
Heyo Mom! Heyo Mom! Mom!
```
rather than this:
```
                           Heyo Mom!
Heyo Mom!
Mom!
```